bin/multisign

```diff
@@ -0,0 +1,218 @@
+#!/usr/bin/env python
+
+# From a single IPA, generate multiple re-signed IPAs simultaneously.
+# Why? Because you might want to distribute an app to a lot of organizations at once,
+# or perhaps you need to sign for an enterprise and a local debug deployment all at
+# the same time, and you want it to be fast.
+
+# Depends on the existence of external `zip` and `unzip` programs.
+
+import argparse
+from os.path import abspath, basename, dirname, expanduser, isdir, join
+from isign import isign
+from isign.archive import archive_factory
+from isign.signer import Signer
+import logging
+import multiprocessing
+
+FORMATTER = logging.Formatter('%(message)s')
+log = logging.getLogger(__name__)
+
+MAX_PROCESSES = multiprocessing.cpu_count() - 1
```

Comment:
Now that you mention it, things might be better if I didn't include that.

What I really would like is to avoid using the same CPU that resto is running on. I don't know if that's easy.